### DISCUSSION TITLE
旧的vue项目可以直接转成Taro项目吗？

### DISCUSSION THREAD

**Discussion by ifour14** [NervJS/taro#12885] [Q & A] [answered]
旧的vue项目可以直接转成Taro项目吗？ 如果可以，具体怎么转？复杂程度怎么样？

**Comment by AdvancedCat** [comment, marked as answer]
可以与否 & 复杂度 取决于以下因素：

1. 旧vue项目大小，是否超过小程序规定包大小；
2. 项目整理：小程序项目规范；taro vue 插件；
3. 兼容性处理：
    a. 运行时是否用了小程序不支持的api，如果用了需要另寻方案替代；
    b. 组件库是否可用；
    c. API需要遵循小程序规范；
    d. more...

以上要根据实际项目来看
